### ISSUE TITLE
Remove Hover Closing of notification

### ISSUE THREAD

**Issue by mdszekeres** [sotsera/sotsera.blazor.toaster#46]
I am trying to figure out how to stop the notification from closing after the mouse is hovered over the notification and removed.

I know I can enable Require interaction however that stops the progress bar from showing and stops the Visible duration timer from closing the notification.

**Comment by pebllle** [comment]
Wow, I haven't thought about this library in a long time.
I'm not sure I understood what you're trying to achieve: are you trying to make the toast require interaction but only after a mouse leave? If this is the case I guess that some code modifications will be needed: the relevant code is [here](https://github.com/sotsera/sotsera.blazor.toaster/blob/38fa473d170633e27343833a893a40dc407d96e9/src/Sotsera.Blazor.Toaster/Core/Toast.cs#L40). 

If I'll ever go back to blazor I'll definitely need to update this library: blazor version 3.1.0-preview4.19579.2 :smile:

**Comment by ghidello** [comment]
Wow, I haven't thought about this library in a long time.
I'm not sure I understood what you're trying to achieve: are you trying to make the toast require interaction but only after a mouse leave? If this is the case I guess that some code modifications will be needed: the relevant code is [here](https://github.com/sotsera/sotsera.blazor.toaster/blob/38fa473d170633e27343833a893a40dc407d96e9/src/Sotsera.Blazor.Toaster/Core/Toast.cs#L40).

If I'll ever go back to blazor I'll definitely need to update this library: blazor version 3.1.0-preview4.19579.2 😄

**Comment by mdszekeres** [comment]
No. I am trying to get the toast to stay open when the Visible duration timer has not elapsed. Currently the toast goes away after a mouse leave.

However it might be possible for the code you linked can achieve this goal.
